camelot/admin/action/base.py

```diff
@@ -153,6 +153,7 @@ class Mode(DataclassSerializable):
     name: str
     verbose_name: typing.Union[str, ugettext_lazy]
     icon: typing.Union[Icon, None]
+    checked: typing.Union[bool, None]
```

Comment:
ok, maar dan mag de klasse FilterMode mss gewoon weg

Reply:
Hmm, ik had dit toegevoegd toen ik nog niet wist dat er een FilterMode klasse was.

Het is waarschijnlijk beter om FilterMode serializeerbaar te maken. Deze heeft ook nog een method die Mode niet heeft.

Reply:
weet niet of dat de moeite is, ik denk dat die method niet gebruikt w ...